File: exercicios/arrays/java/programa03/Main.java
// Encontrar o Maior Elemento

package exercicios.arrays.java.programa03;

public class Main {
    public static void main(String[] args) {
        int[] numeros = {10, 20 ,5, 15, 30};
        int maior = numeros[0];

        for (int i = 1; i < numeros.length; i++) {
            if (numeros[i] > maior) {
                maior = numeros[i];
            }
        }
        System.out.println("O maior elemento do array é: " + maior);
    }
}

File: exercicios/arrays/kotlin/programa03/Main.kt
// Encontrar o maior número em um array de números inteiros

fun main() {
    val numeros = arrayOf(9, 40, 30, 66, 68, 98)

    var maior = numeros[0]

    for (numero in numeros) {
        if (numero > maior) {
            maior = numero
        }
    }
    println("O maior número é : $maior")
}